### PR TITLE
@benisgold/ens header

### PR DESCRIPTION
**Pull request by benisgold**
Fixes RNBW-2402

## What changed (plus any additional context for devs)
- Add placeholders for profile avatar/cover photo
- Pass ens search query to ens profile registration page
- Added profile page nav buttons
- Added "loading network fees" text to search page
- Added function that we can use to get primary color from ens avatar to apply to buttons/text

## PoW (screenshots / screen recordings)
https://user-images.githubusercontent.com/15272675/153478845-34199cfd-2b97-4e9c-b397-3839883357ea.mov

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?

### CODE REVIEW

File: src/navigation/RegisterENSNavigator.js [hunk 1]
```diff
@@ -34,9 +34,9 @@ export default function RegisterENSNavigator() {
 
   return (
     <SlackSheet
-      additionalTopPadding
       contentHeight={contentHeight}
       height="100%"
+      removeTopPadding
```
Comment (estebanmino) [inline src/navigation/RegisterENSNavigator.js:39]: maybe this was the cause of image?

File: src/screens/ENSSearchSheet.js [hunk 2]
```diff
@@ -137,18 +139,18 @@ export default function ENSSearchSheet() {
...
-                            registrationCostsData
-                              ?.estimatedTotalRegistrationCost?.display
-                          }{' '}
+                          {` ${registrationCostsData?.estimatedTotalRegistrationCost?.display} `}
                         </Text>
                         with current network fees
                       </Text>
+                    ) : (
+                      <Text color="secondary50" size="16px" weight="bold">
+                        {'Loading network fees...\n'}
```
Comment (estebanmino) [inline src/screens/ENSSearchSheet.js:152]: 💯

File: src/screens/ENSAssignRecordsSheet.js [hunk 3]
```diff
@@ -16,33 +28,60 @@ export default function ENSAssignRecordsSheet() {
     navigate(Routes.ENS_SEARCH_SHEET);
   }, [navigate]);
 
+  const avatarColor =
+    usePersistentDominantColorFromImage('TODO').result || colors.purple;
```
Comment (jxom) [inline src/screens/ENSAssignRecordsSheet.js:32]: not sure if this is temporary, but if we are skipping avatars for phase 1, we probs want to remove this for performance reasons

File: src/screens/ENSSearchSheet.js [hunk 4]
```diff
@@ -65,11 +67,11 @@ export default function ENSSearchSheet() {
 
   const handlePressContinue = useCallback(() => {
     Keyboard.dismiss();
-    navigate(Routes.ENS_ASSIGN_RECORDS_SHEET);
-  }, [navigate]);
+    navigate(Routes.ENS_ASSIGN_RECORDS_SHEET, { name: `${searchQuery}.eth` });
```
Comment (jxom) [inline src/screens/ENSSearchSheet.js:70]: you can remove this, on the main branch, there is an action to set the name in redux

File: src/screens/ENSSearchSheet.js (hunk 2 above)
Comment (jxom) [reply src/screens/ENSSearchSheet.js:152]: dope

File: src/screens/ENSAssignRecordsSheet.js [hunk 5]
```diff
@@ -56,73 +63,110 @@ export default function ENSAssignRecordsSheet() {
     ],
   });
 
+  const avatarRadius = 35;
+
+  const formIsEmpty = Object.values(values).join('');
```
Comment (jxom) [inline src/screens/ENSAssignRecordsSheet.js:68]: Maybe we could move this logic inside of `useENSProfileForm` and call it `isEmpty`?

```tsx
const {
  isEmpty,
  ...
} = useENSProfileForm(...)
```

File: src/screens/ENSAssignRecordsSheet.js [hunk 6]
```diff
@@ -56,73 +63,110 @@ export default function ENSAssignRecordsSheet() {
...
+          <Row>
+            <Box as={ScrollView} flexGrow={1}>
+              <Box
+                background="accent"
+                height={{ custom: 125 }}
+                marginBottom={{ custom: 70 }}
+              >
+                <Cover alignHorizontal="center">
+                  <Box
+                    background="swap"
```
Comment (jxom) [inline src/screens/ENSAssignRecordsSheet.js:83]: I thought that the placeholder cover would be this, minus the "Add Cover" text:

Screen Shot 2022-02-15 at 1 02 16 pm

Wonder if we should do the same with avatar?

**Comment by jxom** [comment]
I think we may not want to make this handle fixed if the sheet is scrollable (like how it is on NFT expanded sheet), because the user will have to scroll up first before they can dismiss the sheet. 

Screen Shot 2022-02-15 at 1 18 37 pm

**Comment by benisgold** [comment]
> I think we may not want to make this handle fixed if the sheet is scrollable (like how it is on NFT expanded sheet), because the user will have to scroll up first before they can dismiss the sheet.
> 
> Screen Shot 2022-02-15 at 1 18 37 pm

oo good point

**Pull request merged**